### DISCUSSION TITLE
Telescope Configuration Not Working

### DISCUSSION THREAD

**Discussion by VictoryPete** [Stellarium/stellarium#2994] [General]
When I press the Configuration Button nothing happens

**Comment by gzotti** [comment]
https://github.com/Stellarium/stellarium/wiki/FAQ#reporting-a-crash

**Reply by VictoryPete** [reply]
XPS Desktop 8950 210-BCCD 1 McAfee Live Safe XPS 12 Month Subscription 525-0036 1 Killer(TM) Wi-Fi 6 1675 (2x2) 802.11ax Wireless and Bluetooth 5.2 555-BGST 1 Premium Support with Onsite Service After Remote Diagnosis, 1 Year 803-1569 1 Premium Support with Onsite Service After Remote Diagnosis, 3 Year Extended 803-1572 1 Premium Support, 4 Years 803-1590 1 Dell.com Order 332-1530 2 Dell Limited Hardware Warranty Extended Year(s) 975-3461 1 No Mouse Requested 570-AAAH 1 If accessories are purchased, they may ship separately 480-AACF 1 US Power Cord 450-AAGO 1 Windows System Driver 631-ADEM 1 NVIDIA(R) GeForce RTX(TM) 3080 Ti 12GB GDDR6X 490-BGZD 1 750W Night Sky Bezel Chassis including optical drive 321-BHGX 1 Tray load DVD Drive (Reads and Writes to DVD/CD) 429-AAXX 1 No Keyboard Requested 580-AACD 1 Custom Configuration 817-BBBB 1 Killer(TM) Wi-Fi 6 1675 (2x2) 802.11ax Wireless and Bluetooth 5.1 Driver 555-BHDQ 1 No speakers 520-AAAO 1 SERI Guide (ENG/FR/Multi) 340-AGIK 1 Shipping Material 328-BEMM 1 CMS Essentials DVD no Media 658-BBTV 1 Intel(R) Core(TM) i9 non-vPro label 389-DYKH 1 Additional Software 658-BCUJ 1 No Microsoft Office License Included – 30 day Trial Offer Only 658-BCSB 1 McAfee(R) 30day Trial 658-BCCO 1 12th Gen Intel(R) Core(TM) i9-12900K processor (16-Core, 30MB Cache, 3.2GHz to 5.2GHz ) 338-CCQR 1 Windows 11 Pro, English 619-APTO 1 Performance CPU liquid cooling 321-BHGN 1 128GB DDR5, 4x 32GB, at 3600MHz 370-AGSW 1 2TB M.2 PCIe NVME SSD (Boot) + 2TB SATA 7200RPM HDD (Storage) 400-BNCT 1 Dell Limited Hardware Warranty Initial Year
